### ISSUE TITLE
Farm trees BETTER

### ISSUE THREAD

**Issue by TheK0tYaRa** [cabaletta/baritone#1120]
## Describe your suggestion
Use #sel to mark the area for growing trees with sapplings already ready to be bonemealed
Use `#sel farm <sappling name> (<amount of materials and material to farm with multiusage ability>)` to start farming.

## Settings
```
#set
        destroyLeaves <bool>
        rePlaceSapplings <bool>
```

## Context
It automates wood, apple farming. With mods that add trees it would be able to do a lot more than that without having additional modifications to the baritone..

## Final checklist
- [quack] I have not used any OwO's, T_T's or UwU's in this issue.

**Comment by SWEDCODER** [comment]
Yeah, it should definitely be a feature ! #sel is making **everything easier**.

**Comment by lingyunQAQ** [comment]
Hello, I'd like to say in advance that I use machine translation. If there is any wrong statement, I'm very sorry.
My question is, I want to know what to input directly, because I use machine translation and can't understand what you mean. Would you please give a paragraph of instruction directly? Thank you very much.

**Comment by PacoMC-Servicraft** [comment]
Como lo instalo

**Comment by HoguXD** [comment]
I need this to be added
